### PR TITLE
rewrite on 2023.03.02

### PR DESCRIPTION
**Pull request by k-konami**
rewrite on 2023.03.02 (same POT)

### CODE REVIEW

**Review by cooljeanius — APPROVED** [review]

File: translations/wesnoth-The_Earths_Gut/ja.po [hunk 1]
```diff
@@ -2167,8 +2167,8 @@ msgid ""
 "in fact the Hammer of Thursagan, the two of them decided to bring it to "
 "Relgorn."
 msgstr ""
-"Dulatus の取り返したものが 真のThusagan の槌であることを Hamel が確認した後、"
-"二人は Relgorn の許へ持っていくことに決めました。"
+"Dulatus の取り返したものが 真の Thusagan の槌であることを Hamel が確認した"
```
Comment (cooljeanius) [inline translations/wesnoth-The_Earths_Gut/ja.po:2170]: oops I merged this before noticing the typo: missing "r" in "Thursagan"

File: translations/wesnoth-The_Earths_Gut/ja.po [hunk 2]
```diff
@@ -10432,15 +10440,16 @@ msgid ""
 "Wow, the heat has grown so strong that even the flames are coming alive here!"
 msgstr "おー、この熱さは、ここに炎が巻き起こるくらい強力になってきてるぞ！"
 
+# [ja] second_unit?  genderless
```
Comment (cooljeanius) [inline translations/wesnoth-The_Earths_Gut/ja.po:10443]: yes, the unit speaking here can be any gender; the lich's gender we don't know

File: translations/wesnoth-The_Earths_Gut/ja.po (hunk 1 above)
Comment (k-konami) [reply translations/wesnoth-The_Earths_Gut/ja.po:2170]: Ah..... I'm very sorry.   What can I do?

Comment (cooljeanius) [reply translations/wesnoth-The_Earths_Gut/ja.po:2170]: Just be sure to add the missing "r" in your next pull request

Comment (k-konami) [reply translations/wesnoth-The_Earths_Gut/ja.po:2170]: Are the following steps OK?
If ok, then I will do it immediately.
 step1:  Sync my fork to your master
 step2:  fix forked po file, and pull-req again.

**Pull request merged**